UndertaleModLib/Models/UndertaleEmbeddedTexture.cs

```diff
@@ -52,7 +58,13 @@ public void Serialize(UndertaleWriter writer)
         if (writer.undertaleData.GeneralInfo.Major >= 2)
             writer.Write(GeneratedMips);
         if (writer.undertaleData.GM2022_3)
-            writer.Write(TextureBlockSize);
+        {
+            // Write a placeholder for the texture blob size,
+            // so we can overwrite this with the actual value
+            // later
```

Comment:
Can you put the later word on the previous line?

Reply:
```suggestion
test
```